modules/ROOT/pages/mtf.adoc

```diff
@@ -0,0 +1,20 @@
+= Module Testing Framework (MTF)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Module Testing Framework (MTF) enables you to create tests for modules built with Mule SDK for Java or XML to ensure the interaction between modules and Mule runtime is compatible with the Mule ecosystem. MTF supports functional tests for your modules. These tests are run during the `integration-test` phase of your module's lifecycle. 
```

Comment:
Active voice?

You run these tests during...

Reply:
I'm (recently) discouraging writers from using "xxx enables you" after reading this excellent [blog post](https://www.knowledgeowl.com/blog/posts/customer-first-in-your-sentences) from Marcia R. Johnston.